src/styles/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    & :last-child {
```

Comment:
There is an extra space between '&' and ':last-child'. It should be '&:last-child' to correctly target the last child of the stars within the BEM methodology.